cms/djangoapps/contentstore/models.py

```diff
@@ -98,6 +97,11 @@ class EntityLinkBase(models.Model):
     downstream_usage_key = UsageKeyField(max_length=255, unique=True)
     # Search by course/downstream key
     downstream_context_key = CourseKeyField(max_length=255, db_index=True)
+    # This is present if the creation of this link is a consequence of
+    # importing a container that has one or more levels of children.
+    # This represents the parent (container) in the top level
+    # at the moment of the import.
+    top_level_parent_usage_key = UsageKeyField(max_length=255, null=True, blank=True, db_index=True)
```

Comment:
Do we need `null=True`? Django convention is to [never use `NULL` for string fields](https://docs.djangoproject.com/en/5.2/ref/models/fields/#django.db.models.Field.null).

Reply:
Also, did you consider making this a `ForeignKey` to `ContainerLink` instead of a string ID? That would help it stay up to date more consistently.